index.html

```diff
@@ -121,7 +135,9 @@ <h3 class="project_title">Profesional Art Printing Data</h3>
         </ul>
         <a href="#" class="project_link" rel="noopener">See Project</a>
       </div>
+    </div>
   </section>
+  <!-- About me -->
   <section class="about_me">
```

Comment:
- [ ] Please kindly provide the proper background image for this section and align the contents of your about me section as can be seen in the figma template below. It is important that your design matches correctly the provided figma template as this shows your attention to details as a frontend developer and this is a good skill to have :+1: :sparkles: 
- figma design
![ab-me](https://github.com/ommarz/Personal_Portfolio/assets/89055502/b92e3641-cf0f-4d96-9112-fc2a7702669b)
- your design
![abme](https://github.com/ommarz/Personal_Portfolio/assets/89055502/eca53388-51b2-42e3-85bd-0455ffb13811)